examples/mnist/train_mnist_dual_parallel.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
Could you add a short description of what this example demonstrates?

Reply:
I added some explanation in README.